config/clients/go/template/docs/opentelemetry.md.mustache

```diff
@@ -0,0 +1,128 @@
+
+## Metrics
+
+### Supported Metrics
+
+| Metric Name                      | Type      | Enabled by Default | Description                                                                       |
+| -------------------------------- | --------- | ------------------ | --------------------------------------------------------------------------------- |
+| `fga-client.request.duration`    | Histogram | Yes                | Total request time for FGA requests, in milliseconds                              |
+| `fga-client.query.duration`      | Histogram | Yes                | Time taken by the FGA server to process and evaluate the request, in milliseconds |
+| `fga-client.credentials.request` | Counter   | Yes                | Total number of new token requests initiated using the Client Credentials flow    |
+
+### Supported Attributes
+
+| Attribute Name                 | Type   | Enabled by Default | Description                                                                       |
+| ------------------------------ | ------ | ------------------ | --------------------------------------------------------------------------------- |
+| `fga-client.request.client_id` | string | Yes                | Client ID associated with the request, if any                                     |
+| `fga-client.request.method`    | string | Yes                | FGA method/action that was performed (e.g., Check, ListObjects) in TitleCase      |
+| `fga-client.request.model_id`  | string | Yes                | Authorization model ID that was sent as part of the request, if any               |
+| `fga-client.request.store_id`  | string | Yes                | Store ID that was sent as part of the request                                     |
+| `fga-client.response.model_id` | string | Yes                | Authorization model ID that the FGA server used                                   |
+| `fga-client.user`              | string | No                 | User associated with the action of the request for check and list users           |
+| `http.client.request.duration` | int    | No                 | Duration for the SDK to complete the request, in milliseconds                     |
+| `http.host`                    | string | Yes                | Host identifier of the origin the request was sent to                             |
+| `http.request.method`          | string | Yes                | HTTP method for the request                                                       |
+| `http.request.resend_count`    | int    | Yes                | Number of retries attempted, if any                                               |
+| `http.response.status_code`    | int    | Yes                | Status code of the response (e.g., `200` for success)                             |
+| `http.server.request.duration` | int    | Yes                | Time taken by the FGA server to process and evaluate the request, in milliseconds |
```

Comment:
This should be false by default, same for `url.scheme`, `url.full` and `http.request.method` 🤔


But those are less important - I'm not sure if the config matches the documentation, but it is crucial we do not enable `http.server.request.duration` as a default attribute b/c it has a very high cardinality

(Side-note: does not need to be in this PR)

Reply:
Pushed 08996ce202b49254dcf4b9d059d9f1a645a17c61 to update the file with the table/preamble currently used on `main` in go-sdk.